tasks/send-pan/src/index.ts

```diff
@@ -41,16 +41,40 @@ async function sendPAN(event: HandlerEvent): Promise<HandlerOutput> {
   const { config, input } = event;
   const provider = config.provider;
   const remoteDir = config.remoteDir || 'pans';
+  const panType = config.panType || 'shortPan';
 
   const panName = input.pdr.name.replace(/\.pdr/gi, '.pan');
```

Comment:
Any chance you can make this file-extension uppercase? The spec/ICD says the extension is supposed to be .PAN.

Reply:
I changed it to uppercase